Library/Homebrew/cmd/info.rb

```diff
@@ -215,19 +215,15 @@ def github_remote_path(remote, path)
   end
 
   def github_info(f)
-    if f.tap
-      if remote = f.tap.remote
-        path = if f.class.superclass == Formula
-          f.path.relative_path_from(f.tap.path)
-        elsif f.is_a?(Cask::Cask)
-          f.sourcefile_path.relative_path_from(f.tap.path)
-        end
-        github_remote_path(remote, path)
-      else
-        f.path
-      end
-    else
+    if f.tap.nil? && f.tap&.remote.nil?
```

Comment:
This doesn't make sense; if `f.tap.nil?` then `f.tap&.remote` is a pointless check because it'll always be `nil` too. I'd suggest keeping the ordering the same (rather than flipping the `if`/`else`) and, if better, use an early `return`.